packages/devtools_app/lib/src/analytics/analytics_stub.dart

```diff
@@ -0,0 +1,23 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
```

Comment:
can we add more granularity here? I can imagine we'll want to slice data at multiple levels (e.g. screen - feature - action, or screen - feature - subFeature - value).

maybe a generic `track` function with multiple slices? or `trackAction` and `trackValue`?

something like `selectFrame` could then be rolled into the `track` method

Reply:
We can easily add more functions or change the APIs in this stub later on. It's private so should be very easy to do.